int8 command  # 0=关双闪, 1=开双闪, 2=转向灯序列
---
bool success